client/my-sites/upgrades/domain-management/domain-connect/domain-connect-not-found-error.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import EmptyContentComponent from 'components/empty-content';
+
+function DomainConnectNotFoundError( { translate } ) {
+	const emptyContentTitle = translate( 'Uh oh. That method isn\'t supported.', {
+		comment: 'Message displayed when requested Domain Connect URL path is not supported',
+	} );
+	const emptyContentMessage = translate( 'Check with the service provider sent you here for more information.', {
```

Comment:
Seems like something is missing in the message (ugh, I eat up words all the time too).
> Check with the service provider sent you here for more information.
> Check with the service provider *that* sent you here for more information.